docs/designers-developers/developers/data/data-core-edit-post.md

```diff
@@ -372,4 +372,15 @@ Returns an action object used to request meta box update.
 
 ### metaBoxUpdatesSuccess
 
-Returns an action object used signal a successful meta box update.
\ No newline at end of file
+Returns an action object used signal a successful meta box update.
+
+### init
```

Comment:
I might recommend `initialize`, both for (a) alignment to the `wp.editor.initializeEditor` function and (b) to avoid abbreviations which aren't always obvious to the reader.

Reply:
Yes, that makes sense.  I only went with `init` because it matched the original side-effect name `INIT`.